test/e2e/terraform/packer/consul-ami/scripts/datadog.yaml

```diff
@@ -0,0 +1,38 @@
+#########################
+## Basic Configuration ##
+#########################
+
+## @param api_key - string - required
+## The Datadog API key to associate your Agent's data with your organization.
+## Create a new API key here: https://app.datadoghq.com/account/settings
+#
+tags:
+  - "consul_cluster: 173ent-catalog"
```

Comment:
Should this be a variable instead of hard coded?

Reply:
Yeah! I didn't know if Packer could template files that weren't.. explicitly it's own. TIL